payments-model/src/main/java/com/stripe/android/model/LinkAccountSession.kt

```diff
@@ -0,0 +1,12 @@
+package com.stripe.android.model
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.core.model.StripeModel
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class LinkAccountSession(
+    val id: String,
+    val clientSecret: String,
+) : StripeModel
```

Comment:
You can probably use [this model](https://github.com/stripe/stripe-android/blob/master/payments-core/src/main/java/com/stripe/android/model/FinancialConnectionsSession.kt#L13-L12) that already exists.